lib/datadog/statsd.rb

```diff
@@ -432,17 +434,28 @@ def send_stats(stat, delta, type, opts=EMPTY_OPTIONS)
 
     def send_stat(message)
       if @batch_nesting_depth > 0
+        message_bytes = message.bytesize
+        unless @buffer_bytes.zero?
```

Comment:
IMO `if @buffer_bytes != 0` would be much clearer. Admittedly, this is subjective.

Reply:
yeah, updated